plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/AbstractIcebergTableOperations.java

```diff
@@ -59,7 +59,8 @@
 {
     private static final Logger log = Logger.get(AbstractIcebergTableOperations.class);
 
-    protected static final String METADATA_FOLDER_NAME = "metadata";
+    public static final String METADATA_FOLDER_NAME = "metadata";
```

Comment:
why do we need the constants here (instead of `IcebergUtil`) ?

Reply:
This constant was already there with `protected` qualifier. Made it public. Let me know if we can move thses constants to `IcebergUtil`

Reply:
Moved constants to IcebergUtil